From f10f5ff581b73939483ed2a5f78055080332830b Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Tue, 29 Sep 2026 17:29:09 +0200
Subject: [PATCH 137/138] rpcd: update to Git HEAD (2026-09-25)

4062f666ecd1 file: reply with empty data instead of error for empty files
a6c6b63b47bd ucode: add compatibility to newer ucode version

Signed-off-by: Felix Fietkau <nbd@nbd.name>
(cherry picked from commit 34aa4c81eca6179840f7d07500abad56c4ca8a1f)
Link: https://github.com/openwrt/openwrt/pull/25608
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 package/system/rpcd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/system/rpcd/Makefile b/package/system/rpcd/Makefile
index a08ac888d0..e1ab4be6c0 100644
--- a/package/system/rpcd/Makefile
+++ b/package/system/rpcd/Makefile
@@ -12,9 +12,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/rpcd.git
-PKG_MIRROR_HASH:=ca2498a25c69e6b6e8fe1bcc5865a244e409674b1ac2e2e36c0e4bb04b11aba8
-PKG_SOURCE_DATE:=2026-07-19
-PKG_SOURCE_VERSION:=e37ed9d814699098eb7e26c8b33c054840782dfb
+PKG_MIRROR_HASH:=c07273d63b99a8f05fb4a99d1113908a066dd2ff3151662b1fb7ef2ab7c1828c
+PKG_SOURCE_DATE:=2026-09-25
+PKG_SOURCE_VERSION:=a6c6b63b47bd8f4f762ef2e6787210cd3e0fb8a0
 PKG_MAINTAINER:=Jo-Philipp Wich <jo@mein.io>
 
 PKG_LICENSE:=ISC
-- 
2.34.1

